{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2020-4689",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/186696",
					"name": "ibm-guardium-cve20204689-code-exec (186696)",
					"refsource": "XF",
					"tags": []
				},
				{
					"url": "https://www.ibm.com/support/pages/node/6346884",
					"name": "https://www.ibm.com/support/pages/node/6346884",
					"refsource": "CONFIRM",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM Security Guardium 11.2 is vulnerable to CVS Injection. A remote privileged attacker could execute arbitrary commands on the system, caused by improper validation of csv file contents. IBM X-ForceID: 186696."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2020-10-12T14:15Z",
	"lastModifiedDate": "2020-10-13T10:55Z"
}